packages/react-native/lib/trustx-biometric-plugin.test.ts

```diff
@@ -0,0 +1,336 @@
+import {
+  createTrustXIDVProviderFactory,
+  TrustXIDVConfig,
+} from './trustx-biometric-plugin';
+import { IDV_EVENTS, IDVProvider } from '@docknetwork/wallet-sdk-core/src/biometric-provider';
+import axios from 'axios';
+import * as Keychain from 'react-native-keychain';
+import { EventEmitter } from 'stream';
+
+// Mock external dependencies
+jest.mock('axios');
+jest.mock('react-native-keychain', () => {
+  return {
+    getSupportedBiometryType: () => 'FACE_ID',
+    ACCESS_CONTROL: {
+      BIOMETRY_ANY: 1,
+    },
+    ACCESSIBLE: {
+      WHEN_UNLOCKED: 1,
+    },
+    getGenericPassword: jest.fn().mockResolvedValue('data'),
+  };
+});
+
+// Setup mocked axios
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+// Mock wallet
+const mockWallet = {
+  getNetworkId: jest.fn().mockReturnValue('testnet'),
+} as any;
+
+// Sample credentials for testing
+const mockEnrollmentCredential = {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://schema.truvera.io/biometric-context/v1"
+  ],
+  "id": "did:dock:enrollment123",
+  "type": ["VerifiableCredential", "BiometricEnrollment"],
+  "issuer": "did:dock:issuer123",
+  "issuanceDate": "2023-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:dock:wallet123",
+    "biometric_enrollment_id": "enrollment-123",
+    "created": "2023-01-01T00:00:00Z"
+  },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "proofValue": "mock-proof-value"
+  }
+};
+
+const mockMatchCredential = {
```

Comment:
this is the credential being issued by Daon? Do we need to worry about them changing the schema ever?